openapi-validator/src/main/java/io/ballerina/openapi/validator/ServiceValidator.java

```diff
@@ -406,12 +419,22 @@ private DiagnosticSeverity extractOpenAPIAnnotation(DiagnosticSeverity kind, Fil
     private static List<String> setFilters(ListConstructorExpressionNode list) {
         SeparatedNodeList<Node> expressions = list.expressions();
         Iterator<Node> iterator = expressions.iterator();
-        List<String> tags = new ArrayList<>();
+        List<String> values = new ArrayList<>();
         while (iterator.hasNext()) {
             Node item = iterator.next();
-            tags.add(item.toString());
+            if (item.kind() == SyntaxKind.STRING_LITERAL) {
+               Token stringItem = ((BasicLiteralNode) item).literalToken();
+               String text = stringItem.text();
+                if (text.length() > 1 && text.charAt(text.length() - 1) == '"') {
+                    text = text.substring(1, text.length() - 1);
+                } else {
+                    // Missing end quote case
+                    text = text.substring(1);
```

Comment:
Why do we only have a missing end quote condition? can't there be a `missing open quote` condition too?

Reply:
Yeah, it can't be a missing open quote since in ballerina lexer parser has identified the `string` when it has started with the first quotes. If there is no first quote, then it will take the first part as an identifier.  So that time they will provide the compilation error. :)